packages/manager/modules/nutanix/src/dashboard/routing.js

```diff
@@ -19,6 +19,8 @@ export default /* @ngInject */ ($stateProvider) => {
         NutanixService.getServiceDetails(serviceInfo.serviceId),
       nutanixPlans: /* @ngInject */ (user, NutanixService) =>
         NutanixService.getNutanixPlans(user.ovhSubsidiary),
+      newClusterType: /* @ngInject */ (NutanixService, serviceInfo) =>
+        NutanixService.isNewCluster(serviceInfo.serviceId),
```

Comment:
Please rename to `isNewCluster`
```suggestion
      isNewCluster: /* @ngInject */ (NutanixService, serviceInfo) =>
        NutanixService.isNewCluster(serviceInfo.serviceId),
```

Reply:
Done Yann